### PR TITLE
153 Overcooked Integration

### LINKED ISSUE #153: Overcooked implementation
Integrate the overcooked gym environment in cogment-verse.

### PR DESCRIPTION
**Pull request by wduguay-air**
### Context
Overcooked is a great environment to test RL agents. This integration allows to use the versatility and features of Cogment with the gym environment and visualization of the [overcooked_ai](https://github.com/HumanCompatibleAI/overcooked_ai) repository.

- Instructions to setup and play experiments are provided under `doc/integration/overcooked_ai.md`.

#### Other changes
- Added run parameters for render width. Useful for integrations having smaller or larger visual representations.
- Fix serialization issue of MultiBinary spaces encountered in environment integrations.
- Random actor seed based on actor index, otherwise two random actors in a single environment would take the exact same actions.

closes #153

### CODE REVIEW

File: cogment_verse/specs/spaces_serialization.py [hunk 1]
```diff
@@ -37,9 +37,9 @@ def serialize_gym_space(gym_space, serilization_format=SerializationFormat.STRUC
         if isinstance(gym_space.n, np.ndarray):
             size = gym_space.n
         elif isinstance(gym_space.n, int):
-            size = np.array([gym_space.n], dtype=np.dtype("int8"))
```
Comment (wduguay-air) [inline cogment_verse/specs/spaces_serialization.py:40]: Serialization fix from other environment integration for MultiBinary gym spaces.

File: cogment_verse/web/web_app/src/pages/Play.jsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ const Play = () => {
   const redirectToPlayAny = useCallback(() => navigate(".."), [navigate]);
 
   return (
-    <div className="max-w-screen-md mx-auto min-h-screen justify-center">
```
Comment (wduguay-air) [inline cogment_verse/web/web_app/src/pages/Play.jsx:62]: Very tiny changed that fixed a very annoying detail of the web app for a long time. Playing a game with up and down arrows would slightly scroll up and down the screen during a trial, even if the render and instructions perfectly fitted vertically within the window.

File: runs/play.py [hunk 3]
```diff
@@ -109,7 +109,7 @@ def create_trial_params(trial_idx):
                                 actor_config_template=actor_params.get("agent_config", None),
                                 run_id=run_session.run_id,
                                 environment_specs=self._environment_specs.serialize(),
-                                seed=(self._cfg.seed + actor_idx) * trial_idx,
+                                seed=(self._cfg.seed + actor_idx) * trial_idx + actor_idx,
```
Comment (wduguay-air) [inline runs/play.py:112]: Needs this in order for two random actors in the same game to have different actions. I think we should simply attribute specific seeds to each random actor in their configs but it wasn't done in this PR.

Comment (cloderic) [reply runs/play.py:112]: yep it makes sense

Comment (cloderic) [reply runs/play.py:112]: wait no it doesn't the actor_idx is already in the mix

Comment (wduguay-air) [reply runs/play.py:112]: Sorry for the bad explanation, this is for the edge case of trial_idx=0 (the first trial).

Comment (cloderic) [reply runs/play.py:112]: ok then may the fix should be to use something like `self._cfg.seed * (trial_idx + 1) * (actor_idx + 1)` as it is right now the seed is ignored for the first trial

**Review by cloderic — APPROVED** [review]

**Review by cloderic — APPROVED** [review]

**Pull request merged**